packages/cli/src/Init.ts

```diff
@@ -110,6 +131,9 @@ export class Init implements Command {
     
              -h, --help   Display this help message
   --datasource-provider   Define the datasource provider to use: postgresql, mysql, sqlite, sqlserver, mongodb or cockroachdb
+   --generator-provider   Define the generator provider to use: by default, it will be prisma-client-js
+      --preview-feature   Define the preview features to use: by default, there will be none
```

Comment:
Adapting wording a bit more to match the fact this can be used multiple times:
```suggestion
      --preview-feature   Define a preview feature to use: by default, there will be none
```